Volha_Ivanova/Module_1_Java/Java_Fundamentals/HomeWork1/src/com/epam/cdp/volha_ivanova_task1/Runner.java

```diff
@@ -0,0 +1,35 @@
+package com.epam.cdp.volha_ivanova_task1;
+
+public class Runner {                       //Volha Ivanova. В массиве целых чисел определить сумму элементов, состоящих на чётных позициях
+
+	int [] arr1;
+			
+	public static void main(String[] args) {
+    int [] arr1 = new int [20];             // declare massive and its length
+    int min = -10;
+    int max = 10;
+    int j = min;
+    
+   System.out.println("Array elements: \n");
+   
+    for (int i=0; i<arr1.length; i++) {     // filling massive
```

Comment:
Look at your comment. You write the comment to make the code more understandable. But it's bad style to leave comments for the purpose. Honestly comments in code is bad style at all. For comments we use javadocs if it's needed.
In your case you should move your fors blocks to 2 separate methods like 
`private static void createArray(arguments)
{
code
}`

Additionally - filling of array shouldn't print the elements. If you want to print them - please create separate method. Single responsibility principle.